Formula/solo2-cli.rb

```diff
@@ -0,0 +1,31 @@
+class Solo2Cli < Formula
+  desc "Solo 2 library and CLI in Rust"
+  homepage "https://docs.rs/solo2"
+  url "https://github.com/solokeys/solo2-cli/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "ae9ef9dd174a8b8294941635a3a66dc9062fd4b595d5f1f6507b5a5a232d6932"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
```

Comment:
Where is this used?

Reply:
you are right
May some copy paste mistake